app/package.json

```diff
@@ -3,6 +3,7 @@
   "scripts": {
     "build": "vite build",
     "dev": "vite",
+    "container-dev": "vite --host --port 5713",
```

Comment:
I think we can just give these options to the regular `dev` script and avoid the duplication. I can't think of a vite flag that we'd need for the docker container but that would break things if trying to run just the frontend in isolation. Plus, once this PR is merged, I'd think doing `run_stack` would be the preferred way to run things.

Reply:
If you do make this change, can you throw in the `--open` flag if you remember. Won't do anything when running the docker stack, but I'll probably still run the frontend directly often while developing the app and it'll help me there.

Reply:
You're right, just adding them to `bun run dev` command in the Dockerfile seemed to work fine; I could've sworn I tried that, failed, and resorted to adding an entry in `package.json`, but I guess I must have misremembered or done it incorrectly.

About the `--open` flag, I didn't think it would work from within a Docker container, but I tried it anyway and as I suspected it didn't open a browser window. I tried a few options with similar behavior to `vite --open`, but unfortunately none were a perfect fit; I'll keep thinking about the issue, but for now I'll just use `open` on OS X, even though it always opens a new tab.